docs/rules/visual-label-better-than-aria-suggestion-v9.md

```diff
@@ -0,0 +1,29 @@
+# Accessibility: Visual labels should be preferred over aria-label (`@microsoft/fluentui-jsx-a11y/visual-label-better-than-aria-suggestion-v9`)
```

Comment:
I think there are still linting issues on the readme file :)

Reply:
I tried running the md generator command but it's changing some random files and my file was passing lint when trying locally. @aubreyquinn can you please indicate the issue if you can figure out in this readme file